tests/aws/services/lambda_/test_lambda_destinations.py

```diff
@@ -603,13 +607,9 @@ def test_invoke_lambda_eventbridge(self, infrastructure, aws_client, snapshot):
         wait_until_log_group_exists(triggered_fn_name, aws_client.logs)
 
         def _filter_message_triggered():
-            log_events = aws_client.logs.filter_log_events(
-                logGroupName=f"/aws/lambda/{triggered_fn_name}"
-            )["events"]
-            filtered_logs = [event for event in log_events if event["message"].startswith("{")]
-            assert len(filtered_logs) >= 1
-            filtered_logs.sort(key=lambda e: e["timestamp"], reverse=True)
-            return filtered_logs[0]
-
-        log = retry(_filter_message_triggered, retries=50 if is_aws_cloud() else 10)
-        snapshot.match("filtered_message_event_bus_sqs", log["message"])
+            log_events = get_lambda_log_events(triggered_fn_name, logs_client=aws_client.logs)
```

Comment:
Daniel doesn't recommend this, we should fix this in a follow up and match the whitespace offering full AWS parity